dev-portal/public/custom-content/content-fragments/GettingStarted.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Getting Started
+---
+
+## Getting Started
+
+### Create an account and subscribe to APIs
+
+To use any of our APIs you must have a developer account. A developer account gives you an API Key, access to use and test our APIs, and metrics on your usage. Create or sign in using the buttons in the top right.
+
+When you first register, your new API Key won't be tied to any of our APIs. To activate it for an API, navigate to [APIs](/apis), then find the API you want to use and click `subscribe`. Your API Key is now subscribed to the API and you can make calls to all methods on the API.
+
+Figuring out how to use APIs can be hard. Use the “Try it out!” feature to get examples of the request and response shapes of our APIs. The Developer Portal will make an API call to the backend service using your API key and shows a sample `curl` request with all necessary input parameters and the real response.
+
+If you need your API Key for any reason, you can always find it on [your dashboard](/dashboard) after logging in.
+
+### Monitor your usage
+
+Typically each API has a usage limit set for each API Key. As you scale up your usage of the APIs, you can monitor your usage towards the limits on [your dashboard](/dashboard).
```

Comment:
Are we consuming the usage limits yet to draw lines on the graphs for the "cap"?